databricks/koalas/namespace.py

```diff
@@ -1635,6 +1626,23 @@ def _to_datetime2(arg_year, arg_month, arg_day,
         format=format,
         infer_datetime_format=infer_datetime_format)
 
+    index_map = None  # type: Optional[List[IndexMap]]
```

Comment:
What's this for?

Reply:
Oops maybe i added it by mistake. i'll fix it. 😭